src/components/ProfileCard/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+interface ProfileCard {
+  readonly imagePath: string
+  readonly alternate: string
+  readonly name: string
+  readonly role: string
+}
+
+const ProfileCard = ({ imagePath, alternate, name, role }: ProfileCard) => {
+  return (
+    <div className="profile-card profile-card--background-image">
+      <div className="profile-card__body">
+        <img src={imagePath} alt={alternate} className="profile-card__image" />
```

Comment:
Currently the `margin` is applied on the `img`, would be better to have an `image-container`.